.tools/readme.py

```diff
@@ -0,0 +1,20 @@
+import re
+
+with open("README.md") as fh:
+    readme = fh.read()
+
+with open("src/changelist/default_config.toml") as fh:
+    default_config = fh.read()
+
+config_begin = r"<!--- begin default_config.toml --->\n"
+config_end = r"<!--- end default_config.toml --->\n"
+config_section = (
+    config_begin + r"\n````toml\n" + default_config + r"````\n\n" + config_end
+)
+
+rx = re.compile(config_begin + ".*?" + config_end, re.DOTALL)
+readme = rx.sub(config_section, readme)
```

Comment:
The solution with looking for HTML comments seems reasonably robust. Another approach might be to simply test
```python
default_config in readme
```

Reply:
And then require the file to be manually fixed?

Reply:
Yes, I originally had the idea of running this as a pytest. But I am happy with the regex.sub-solution.